src/comments/comments.go

```diff
@@ -0,0 +1,69 @@
+package githubComments
```

Comment:
If I am not mistaken, one of Go's convention is that packages should be all lowercase, so `githubComments` should just be `githubcomments` or simply `comments`.

Reply:
sure, good catch!